coderd/workspaceagents.go

```diff
@@ -1463,6 +1473,96 @@ func (api *API) workspaceAgentsExternalAuthListen(ctx context.Context, rw http.R
 	}
 }
 
+// @Summary User-scoped agent coordination
```

Comment:
```suggestion
// @Summary User-scoped tailnet RPC connection
```

I should have been prescriptive the first time.